docs/multihaul.rst

```diff
@@ -0,0 +1,52 @@
+multihaul
+=========
+
+.. dfhack-tool::
+    :summary: Haulers gather multiple nearby items when using wheelbarrows.
+    :tags: fort productivity items stockpile
+
+This tool allows dwarves to collect several adjacent items at once when
+performing hauling jobs with a wheelbarrow. When enabled, new
+``StoreItemInStockpile`` jobs with wheelbarrows will automatically attach nearby items so
+they can be hauled in a single trip. Jobs without wheelbarrows would try to attach one if autowheelbarrows option is on.
+Items claimed by another jobs or already stored in stockpiles would be ignored.
+By default, up to ten additional items within 10 tiles of the original item are collected.
+Warning: Destination stockpile filters are currently ignored by the job (because of DF logic). Which items qualify can be controlled
```

Comment:
I think it would be highly desirable for the tool to respect the stockpile settings. Can you elaborate on what the issue is?